tests/test_other.py

```diff
@@ -8957,6 +8957,30 @@ def test_node_js_run_from_different_directory(self):
     ret = run_process(NODE_JS + [os.path.join('subdir', 'a.js')], stdout=PIPE).stdout
     self.assertContained('hello, world!', ret)
 
+  # Tests that a pthreads + modularize build can be run in node js
+  @no_fastcomp('node pthreads only supported on wasm backend')
+  def test_node_js_pthread_module(self):
+    # create module loader script
+    moduleLoader = 'moduleLoader.js'
+    moduleLoaderContents = '''
+const test_module = require("./module");
+test_module().then((test_module_instance) => {
+  test_module_instance._main();
+  process.exit(0);
+});
+'''
+    if not os.path.exists('subdir'):
+      os.mkdir('subdir')
+    with open(os.path.join('subdir', moduleLoader), 'w+', encoding='utf-8') as f:
```

Comment:
there is a helper function for this, `create_test_file(name, contents)`

Reply:
Made the change, but I'm not sure what's going on with the tests. My test passed, but some other ones appear to have hung.